debian/postinst

```diff
@@ -26,13 +26,20 @@ SYSTEMD_HELPER_ENABLED_AUTO_ATTACH_DSH="/var/lib/systemd/deb-systemd-helper-enab
 SYSTEMD_HELPER_ENABLED_WANTS_LINK="/var/lib/systemd/deb-systemd-helper-enabled/multi-user.target.wants/ua-auto-attach.service"
 
 
+check_esm_infra_is_enabled() {
+    return $(ua status | egrep -q 'esm-infra.*enabled')
+}
+
```

Comment:
Sorry I broke your actually working code with my suggestion, we can't actually return the exit code here, it just returns the stdout.
So this works; just tested:
```suggestioncheck_esm_infra_is_enabled() {
    ua status | egrep -q 'esm-infra.*enabled' && return 0 || return 1
}

```